doc/source/userguide.rst

```diff
@@ -202,26 +185,21 @@ Currently, two backends exist for the 2D scattering transform:
   ``scikit-cuda``. This implementation only runs on the GPU (that is, you must
   call :meth:`cuda` prior to applying it). Since it uses kernels optimized for
   the various steps of the scattering transform, it achieves better performance
-  compared to the default ``torch`` backend (see benchmarks below).
-
-3-D backend
------------
-
-Currently, one backends exists for the 3D scattering transform:
-
-- ``torch``: A PyTorch-only implementation which is differentiable with respect
-  to its inputs. However, it relies on general-purpose CUDA kernels for GPU
-  computation which reduces performance.
+  compared to the default ``torch`` backend (see benchmarks below). This
+  improvement is currently small in 1D and 3D, but work is underway to further
+  optimize this backend.
 
-Benchmark with previous versions
-================================
+Benchmarks
+==========
 
-1-D backend
+1D backend
 -----------
 
 We compared our implementation with that of the ScatNet MATLAB package
 :cite:`anden2014scatnet` with similar settings. The following table shows the
-average computation time for a batch of size :math:`64 \times 1 \times 65536`:
+average computation time for a batch of size :math:`64 \times 65536`. This
+corresponds to :math:`64` signals containing :math:`65536`, or a total of about
```

Comment:
Did we have dollar-math now or not?

Reply:
I guess we do. I was too lazy to check. Will change.